### PR TITLE
Display total and scaled load values if check_load scales the values by number of CPUs

### PR DESCRIPTION
**Pull request by RincewindsHat**
Replaces #1548

### CODE REVIEW

File: plugins/check_load.c [hunk 1]
```diff
@@ -211,9 +206,26 @@ main (int argc, char **argv)
...
+			char *tmp = NULL;
+			xasprintf(&tmp, _("load average: %.2f, %.2f, %.2f"), scaled_la[0], scaled_la[1], scaled_la[2]);
+			xasprintf(&status_line, "scaled %s - %s", tmp, status_line);
+	}
+
 	printf("LOAD %s - %s|", state_text(result), status_line);
-	for(i = 0; i < 3; i++)
-		printf("load%d=%.3f;%.3f;%.3f;0; ", nums[i], la[i], wload[i], cload[i]);
+	for(i = 0; i < 3; i++) {
+		if (scaled_la[i] != 0.0) {
```
Comment (sni) [inline plugins/check_load.c:222]: i don't think it's a good idea to make printing the performance values depend on the actual value. Either you want it or not, but don't magically hide some values if they are zero.

File: plugins/t/check_load.t [hunk 2]
```diff
@@ -11,8 +11,10 @@ use NPTest;
 my $res;
 
 my $loadValue = "[0-9]+\.?[0-9]+";
-my $successOutput = "/^LOAD OK - load average: $loadValue, $loadValue, $loadValue/";
-my $failureOutput = "/^LOAD CRITICAL - load average: $loadValue, $loadValue, $loadValue/";
+my $successOutput = "/^LOAD OK - total load average: $loadValue, $loadValue, $loadValue/";
+my $successScaledOutput = "/^LOAD OK - scaled load average: $loadValue, $loadValue, $loadValue - total load average: $loadValue, $loadValue, $loadValue/";
```
Comment (sni) [inline plugins/t/check_load.t:15]: is this used anywhere?

File: plugins/check_load.c (hunk 1 above)
Comment (RincewindsHat) [reply plugins/check_load.c]: This is completely true, this was the "poor man's version" of testing whether the scaled values are actually in usage or not.
Solved this another way now.

File: plugins/t/check_load.t (hunk 2 above)
Comment (RincewindsHat) [reply plugins/t/check_load.t:15]: It is now :-)

**Review by sni — APPROVED** [review]

**Pull request merged**